terraform/helm/aptos-node/values.yaml

```diff
@@ -31,11 +31,11 @@ haproxy:
     pullPolicy: IfNotPresent
   resources:
     limits:
-      cpu: 1.5
-      memory: 2Gi
+      cpu: 4
```

Comment:
this go beyond the current default utility group can support. can we update the default instance for utility group too?

Reply:
using same group would cause people extra money for running that extra validator instance, which is very expensive for mainnet default